escrow/src/main.sw

```diff
@@ -1,12 +1,15 @@
 contract;
 
+// TODO: commented-out code is there because the SDK does not support Vec yet so an array is used
+
 // Our library dependencies
 dep data_structures;
 dep errors;
 dep events;
 dep interface;
 
 use data_structures::{Arbiter, Asset, Buyer, EscrowInfo, Seller, State};
+// use data_structures::*;
```

Comment:
remove line as I don't think it applies to the TODO at the top of the file

Reply:
This has been resolved in [this commit](https://github.com/FuelLabs/sway-applications/pull/54/commits/b966c5d460bd68af8b564b89a8fb044b3416ce74) a little higher up

Reply:
lol whoops